lisa/features/serial_console.py

```diff
@@ -199,3 +199,6 @@ def write(self, data: str) -> None:
     def close(self) -> None:
         # it's not required to implement close method.
         pass
+
+    def execute_command(self, commands: List[str]) -> str:
```

Comment:
Improve the `_write` instead of creating another similar method.

Reply:
Ack.